server/models/Availability.js

```diff
@@ -0,0 +1,118 @@
+const mongoose = require('mongoose');
+
+const MIN = 0;
+const MAX = 2359;
+
+const time = new mongoose.Schema({
+  hour: {
+    type: Number,
+    min: 0,
+    max: 23
+  },
+  minute: {
+    type: Number,
+    min: 0,
+    max: 60,
+    default: 0
+  },
+})
+
+const range = new mongoose.Schema({
+  from: {
+    type: Number,
+    min: 0,
+    max: 2359,
+    default: 0
+  },
+  to: {
+    type: Number,
+    min: 0,
+    max: 2359,
+    default: 0
+  }
+});
+
+const availabilitySchema = new mongoose.Schema({
```

Comment:
Let's add a `name` property so we can have multiple availabilities per sitter.

Reply:
done